README.md

```diff
@@ -50,9 +58,42 @@ to always keep macOS on and available.
 | `node['macos']['network_time_server']`  | `'time.windows.com'`    |
 | `node['macos']['time_zone']`            | `'America/Los_Angeles'` |
 
+### Xcode
```

Comment:
This was removed in #187

Reply:
Changed by rebasing release/3.0 into this branch and reverting commit.